src/Features/VisualBasic/Portable/UseAutoProperty/VisualBasicUseAutoPropertyCodeFixProvider.vb

```diff
@@ -63,19 +63,30 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
                 statement = statement.WithAsClause(initializer.asNewClause)
             End If
 
+            If initializer.arrayBounds IsNot Nothing Then
+                Dim arrayCreation = SyntaxFactory.ArrayCreationExpression(
+                    Nothing,
+                    DirectCast(statement.GetReturnType(), ArrayTypeSyntax).ElementType,
+                    initializer.arrayBounds,
+                    If(TryCast(initializer.equalsValue?.Value, CollectionInitializerSyntax), SyntaxFactory.CollectionInitializer()))
+                statement = statement.WithTrailingTrivia(SyntaxFactory.Space) _
+                    .WithInitializer(SyntaxFactory.EqualsValue(arrayCreation))
```

Comment:
```suggestion
                statement = statement.WithTrailingTrivia(SyntaxFactory.Space).
                                      WithInitializer(SyntaxFactory.EqualsValue(arrayCreation))
```

Reply:
I'm following the style with several lines above here.